tree/ntuple/v7/inc/ROOT/RColumnElement.hxx

```diff
@@ -315,7 +315,7 @@ public:
 
    std::size_t GetSize() const { return fSize; }
    std::size_t GetBitsOnStorage() const { return fBitsOnStorage; }
-   std::size_t GetPackedSize(std::size_t nElements = 1U) const { return (nElements * fBitsOnStorage + 7) / 8; }
+   virtual std::size_t GetPackedSize(std::size_t nElements = 1U) const { return (nElements * fBitsOnStorage + 7) / 8; }
```

Comment:
I think this inadvertently introduces a `virtual` function call in the hot code path of reading and writing simple column types, something that we really try to avoid. If the return cannot be computed anymore (I didn't check the logic), I think it would be better to introduce a new member field that holds the value directly.